src/app/pages/manage-organisation/manage-organisation-registration-step-1/manage-organisation-registration-step-1.component.scss

```diff
@@ -28,3 +28,11 @@ a{
   font-size: 20px;
 }
 
+.govuk-link:hover {
```

Comment:
i am not sure why we have to use our style here instead of using the style from GDS. Let me approve it now. We will review it later.